<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditKeyDialog</class>
 <widget class="QDialog" name="EditKeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Key</string>
  </property>
  <layout class="QVBoxLayout" name="plLayout">
   <item>
    <widget class="QLabel" name="plName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pleName"/>
   </item>
   <item>
    <widget class="QWidget" name="pwFree" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pwButtons" native="true">
     <layout class="QHBoxLayout" name="plButtons">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="ppbSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pwBtnFree" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="ppbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
